cinema/views.py

```diff
@@ -101,8 +102,27 @@ def get_serializer_class(self):
         if self.action == "retrieve":
             return MovieDetailSerializer
 
+        if self.action == "create":
+            return MovieCreateSerializer
+
+        if self.action == "upload_image":
+            return MovieImageSerializer
```

Comment:
Ensure that the `MovieImageSerializer` is correctly used in the `upload_image` action to handle image uploads as required by the task.